src/components/PeopleList/PeopleList.tsx

```diff
@@ -0,0 +1,26 @@
+import { Person } from '../../types';
+import { PersonInfo } from '../PersonInfo/PersonInfo';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleList: React.FC<Props> = ({ people }) => {
+  return (
+    <>
+      {people.map((person) => {
+        const mother = people.find(human => human.name === person.motherName);
+        const father = people.find(human => human.name === person.fatherName);
```

Comment:
I don't think we want to do it on every render, let's instead prepare array with people after we received data from server